### DISCUSSION TITLE
GitHub login issue

### DISCUSSION THREAD

**Discussion by kovacsbenceckik** [community/community#44554] [Other Feature Feedback, Questions, & Ideas]
Hey guys, I can’t go to the original github.com website on my pc. I can access my account from my phone, but I need access from my pc. Probably my ip is banned. How can I contact the devs about this?

**Comment by idadwind1** [comment]
[contact support](https://support.github.com/contact/reinstatement)

**Comment by gondaljutt** [comment]
Are you using the same network on both devices?

**Comment by idadwind1** [comment]
How does the website react when you try logging in?
